modules/microshift-disabling-uninstalling-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-disabling-uninstalling-lvms-csi-snapshot_{context}"]
+= Disabling and uninstalling LVMS CSI provider and CSI snapshot deployments
+
+You can reduce the use of runtime resources such as RAM, CPU, and storage by removing or disabling the following storage components:
+
+* You can configure {microshift-short} to disable the built-in logical volume manager storage (LVMS) Container Storage Interface (CSI) provider.
+* You can disable the CSI snapshot capabilities.
+* You can uninstall the installed CSI implementations using `oc` commands.
```

Comment:
The CSI snapshot bullets should be matched by the LVMS bullets:
You disable LVMS...
You can uninstall LVMS...

Also, it's probably good to call out that you can only preemptively disable either component before start microshift for the first time. After that you must update the config AND uninstall the components.

Reply:
done